upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.15.yaml.template

```diff
@@ -25,8 +22,12 @@ data:
   # the pod network.
   masquerade: "true"
 
-  # MTU default is 1500, can be overridden
-  veth_mtu: "{{- or .Networking.Canal.MTU "1500" }}"
+  # Configure the MTU to use
+  {{- if .Networking.Canal.MTU }}
+  veth_mtu: "{{ .Networking.Canal.MTU }}"
+  {{- else }}
+  veth_mtu: "{{- if eq .CloudProvider "openstack" -}}1430{{- else -}}1440{{- end -}}"
```

Comment:
If unset I believe it previously defaulted to 1500, should that be 1440 now in all cases?

Reply:
I used the same logic from the Calico side. The value seemed a safer default for most environments, including when IP-in-IP is used (for cross subnet):
https://docs.projectcalico.org/v3.10/networking/mtu#determine-mtu-size

Unless users know what they are doing I thing it is a